packages/sysutils/busybox/scripts/init

```diff
@@ -470,12 +464,12 @@ force_fsck() {
   read -t120 -n1
   # The exit status is 0 if input is available
   # The exit status is greater than 128 if the timeout is exceeded
-  if [ "$?" -ne "0" -o "$?" -gt "128" ] ; then
+  if [ "$?" -ne "0" -o "$?" -gt "128" ]; then
```

Comment:
Side issue, but not sure why we are mixing strings with unary comparisons, shouldn't this be:
```
   if [ $? -ne 0 -o $? -gt 128 ]; then
```
as `$?` will only ever be an integer.